SU2_CFD/include/solvers/CHeatSolver.hpp

```diff
@@ -41,7 +41,10 @@ class CHeatSolver final : public CSolver {
   static constexpr size_t MAXNDIM = 3; /*!< \brief Max number of space dimensions, used in some static arrays. */
   static constexpr size_t MAXNVAR = 1; /*!< \brief Max number of variables, for static arrays. */
 
-  unsigned short nVarFlow, nMarker, CurrentMesh;
+  const bool flow; /*!< \brief Use solver as a scalar transport equation of Temperature for the inc solver. */
+  const bool heat_equation; /*!< \brief use solver for heat conduction in solids. */
```

Comment:
Or you could have given them default values, then the default constructors use those.

Reply:
hm, I would actually prefer to make the new-constructor-author aware that he needs to take care of that ... but on the other hand one might expect that kind of awareness. But that's more of a general idea. Thanks for hint ... didn't have that on my watch

Reply:
Equally valid, just pointing out alternatives :)